kubernetes/helm/pinot/values.yaml

```
@@ -106,6 +106,13 @@ controller:
     port: 9000
     annotations: {}
 
+  ingress:
+    enabled: false
+    annotations: { }
+    tls: []
```

Should be an object:

```suggestion
    tls: {}
```